src/components/article/Comments.jsx

```diff
@@ -65,11 +59,19 @@ function handlePostComment(){
         {commentsVisible ? "Hide Comments" : "View Comments"}
       </button>
       <div className={commentsVisible ? "comments-section" : "hidden"}>
-
-      <p>{isLoading? `Hang tight whilst we load your comments :)`: null}</p>
-      <CommentPost article_id={article_id} handleViewComments={handleViewComments}/>
+        <CommentPost
+          article_id={article_id}
+          handleViewComments={handleViewComments}
+        />
+        <p>{isLoading ? `Hang tight whilst we load your comments :)` : null}</p>
         {comments.map((comment) => {
-          return <CommentCard key={comment.comment_id} comment={comment} />;
+          return (
+            <CommentCard
+              key={comment.comment_id}
+              comment={comment}
+              handleViewComments={handleViewComments}
+            />
+          );
         })}
```

Comment:
<img width="1324" alt="image" src="https://github.com/isaacluke/nc-news-project-fe/assets/151947206/7bfb6290-6e4a-4be5-8b94-4dec61f5e9b2">
Dark mode is really doing a number on me here 🕶️